haiku-misc/quicktour/quicktour-0.2.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A quick tour of Haiku, aimed at new users"
+DESCRIPTION="The Haiku Quick Tour is a set of HTML pages that introduce \
+Haiku's unique features especially to new users. Topics range from window \
+widgets and the Deskbar to window management like Stack & Tile and where \
+to get more software. All in under 20 slides.
+After installation a link 'QuickTour' is created on the Desktop."
+HOMEPAGE="https://github.com/humdingerb/quicktour"
+COPYRIGHT="2018 Humdinger"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/humdingerb/quicktour/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1d99c2efb0e1d466af009e4e079249fbe78cce8cce2335814f3098b632b7be15"
+ADDITIONAL_FILES="quicktour-link-on-desktop.sh"
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/quicktour-link-on-desktop.sh"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	quicktour = $portVersion
+	"
+REQUIRES=""
+
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES=""
+
+BUILD()
+{
+	true
+}
+INSTALL()
+{
+	quickDir=$documentationDir/quicktour
+	mkdir -p $quickDir
+
+	cp -r en $quickDir
+	cp -r images $quickDir
+	cp -a Haiku-tour.css $quickDir
+
+	mkdir -p $postInstallDir
+	cp $portDir/additional-files/quicktour-link-on-desktop.sh $postInstallDir
+	chmod +x $postInstallDir/quicktour-link-on-desktop.sh
```

Comment:
Maybe:
```diff
- 	chmod +x $postInstallDir/quicktour-link-on-desktop.sh
```
Since git is able to store the file permissions, you can `chmod +x haiku-misc/quicktour/additional-files/quicktour-link-on-desktop.sh` and remove the `chmod` in `INSTALL`.

I guess haikuporter will preserve it but I did not check this.

Now, I don't know if this change is desirable.